### ISSUE TITLE
Frontend Option: Bootstrap

### ISSUE THREAD

**Issue by itsthejoker** [GrafeasGroup/tor_app#18]
Epic: #8

This issue exists to discuss the above framework.

Bootstrap:

Pros:
- well-documented
- stable
- large community
- easy to get started

Cons:
- slightly older, slightly dated appearance
- large -- ~270kb

**Comment by itsthejoker** [comment]
This is the option I'm personally most familiar with and can bang out the most code in; even though it has a bit of slightly dated feel to it, it's still fast and easy to run (especially when paired with something like jquery and / or preact).

**Comment by perryprog** [comment]
I personally would rather not use Bootstrap — It has a very similar look across webpages, that often doesn't look that nice.

**Comment by itsthejoker** [comment]
I agree -- something else I forgot to write last night is that it also handles transitions between desktop and mobile much easier.

**Comment by DavidArchibald** [comment]
I'm surprised you didn't mention [themes](https://themes.getbootstrap.com/) in the pros, considering your apparent like of them in discord.

P.S. I linked the paid official ones, but there are free ones.

**Comment by itsthejoker** [comment]
We can have the nonprofit purchase a theme if we wanted to go that route; not a big issue.

**Comment by itsthejoker** [comment]
I quite like this one, and it's free; the Pro version is nicer, but buying a theme for $70 would pretty much blow out our bank account. https://github.com/creativetimofficial/now-ui-kit

**Comment by itsthejoker** [comment]
Someone who was reading this conversation reached out to me privately and offered to pay for part of the cost of the now-ui-kit-pro theme, should we want to go for it. Personally, I think it looks amazing but first we need to decide if bootstrap is indeed right for us.

https://demos.creative-tim.com/now-ui-kit-pro/index.html

**Comment by itsthejoker** [comment]
Okay, let's plan on using Bootstrap 4. I'll keep this open for a bit so we can find out what's happening with the theme; I sent an email to the folks over at Creative Tim to see if they'd be interested or willing in doing a sponsorship (because on further reading I'm not sure if we're supposed to buy the $70 license or the $280 license 😱). We'll see what they have to say.

**Comment by itsthejoker** [comment]
I'm very happy to say that Creative Tim has offered to sponsor us and will be providing a Developer license to UI Kit Pro!
